samples/dungeon/common/src/main/java/com/squareup/sample/dungeon/board/Parser.kt

```diff
@@ -106,7 +103,7 @@ private fun BufferedSource.readHeader(): String? = buildString {
   while (true) {
     val line = readUtf8Line() ?: throw IllegalArgumentException("Expected --- but found EOF.")
     if (line == YAML_DELIMITER) return@buildString
-    appendln(line)
+    appendLine(line)
```

Comment:
We can't use the stdlib from kotlin 1.4 yet, since we can't use kotlin 1.4 internally.

Reply:
☹️